src/mpid/ch4/shm/posix/eager/iqueue/iqueue_init.c

```diff
@@ -24,12 +24,12 @@
     - name        : MPIR_CVAR_CH4_SHM_POSIX_IQUEUE_CELL_SIZE
       category    : CH4
       type        : int
-      default     : 69632
```

Comment:
iqueue cells are also used for bulk data when other transports are not available. Will this size change have an impact? ch3/nemesis uses 64K cells.

Reply:
Smaller cell size means more chunks and it will incur more overhead. However, if it is just an arbitrary threshold, then there is no real difference between 16k and 64k. It is just a slightly different balance of memory and performance at medium message size.

Reply:
I'm not sure 64k is arbitrary. The nemesis code went through some performance study, but I can't find any documentation of the cell size choice unfortunately. This chance is a nak without some measurement. Looking back at the commits, it seems this pack buffer size change is also hidden in the consolidation commit without any mention. That one is also a nak for now.